src/SqlPersistence/SqlPersistenceStorageSessionExtensions.cs

```diff
@@ -15,12 +18,36 @@ public static class SqlPersistenceStorageSessionExtensions
         /// </summary>
         public static ISqlStorageSession SqlPersistenceSession(this SynchronizedStorageSession session)
         {
+            return GetSqlStorageSession(session);
+        }
+
+        internal static StorageSession GetSqlStorageSession(this SynchronizedStorageSession session)
```

Comment:
Inconsistent naming: here `SqlPersistenceSession`, NHIbernate `Session()`, Raven `RavenSession()`

> "Shared session has not been configured for SqlPersistence."

Is that something that users need to configure? How? Exception message doesn't seem to be very helpful.